backend/hitas/views/owner.py

```diff
@@ -2,7 +2,7 @@
 from rest_framework.response import Response
 
 from hitas.models import Owner, Ownership
-from hitas.views.utils import HitasCharFilter, HitasFilterSet, HitasModelSerializer, HitasModelViewSet, HitasSSNFilter
+from hitas.views.utils import HitasCharFilter, HitasFilterSet, HitasModelSerializer, HitasModelViewSet
```

Comment:
Could be squashed with the commit, in which you made changes to the same file

Reply:
Right you are - this was more of a reaction to the tests not passing, and me not being too familiar with the backend. :)